os/net/netmgr/netdev_mgr_internal.c

```diff
@@ -46,7 +46,7 @@ static int g_eth_idx = 0;
  * and application doesn't require write operation to netdev
  * so if there is no write operation then protection wouldn't be required
  */
-sem_t g_netdev_lock = SEM_INITIALIZER(1);
+sem_t g_netdev_lock;
```

Comment:
I think this variable is used only in this file. If I'm right, please add static.